subprojects/build-init/build-init.gradle.kts

```diff
@@ -59,7 +59,7 @@ tasks {
 
             findLatest("scalatest", "org.scalatest:scalatest_${versionProperties["scala"]}:(3.0,)", versionProperties)
```

Comment:
Can we now use a Scala version bigger than `2.12.x`?

Reply:
I'll leave that out of this PR's scope